src/preamble.js

```diff
@@ -130,14 +130,23 @@ function ccall(ident, returnType, argTypes, args, opts) {
     });
   }
 #else // EMTERPRETIFY_ASYNC
+  if (typeof Asyncify === 'object' && Asyncify.currData !== null) {
```

Comment:
`Asyncify.currData !== null` means that asynchronous execution is currently happening.